game/src/character/objects/TreasureChest.java

```diff
@@ -20,16 +20,19 @@ public class TreasureChest extends AnimatableElement implements Collidable {
     private Point currentPosition;
     private ILevel currentLevel;
     private boolean isOpen = false;
-    private int openingAnimationTime = 15;
+    private final int openingFrames = 3;
+    private final int openingFrameTime = 30;
+    private int openingAnimationTime = (int) ((openingFrameTime * openingFrames) / 2);
     protected Hitbox hitbox;
     private List<Item> inventory;
 
     public TreasureChest(Point position) {
-        List<String> texturePaths = TextureHandler.getInstance().getTexturePaths("ui_heart_full");
+        List<String> texturePaths =
+                TextureHandler.getInstance().getTexturePaths("chest_full_open_anim_f0");
         closed = new Animation(texturePaths, 1);
-        texturePaths = TextureHandler.getInstance().getTexturePaths("ui_heart");
-        opening = new Animation(texturePaths, 5);
-        texturePaths = TextureHandler.getInstance().getTexturePaths("ui_heart_empty");
+        texturePaths = TextureHandler.getInstance().getTexturePaths("chest_full_open_anim_");
```

Comment:
Bei mir findet das hier die Animation zweimal, also insgesamt 6 Animationsframes: 

![chest_texture](https://user-images.githubusercontent.com/37513253/201926693-dfe45d2f-1ec4-4ec1-a2ec-f1de01208c45.png)

Mit dem `/2` in `(int) ((openingFrameTime * openingFrames) / 2)` brichst du hier die Animation einfach nach der Hälfte (also nach Frame 3 / Index 2) ab.

Es wird anscheinend auch nicht sichergestellt, dass die Animationsframes in der richtigen Reihenfolge gespeichert werden

Reply:
Das Problem besteht auch bei den anderen texturePaths, da sind dann immer 2 statt einem Frame drin.

Reply:
Klingt nach einem Bug im Framework. Danke. Ich fix das hier mit einem workarround und erstell ein ticket im pm-dungeon